variables.kubernetes.scaffolding.tf

```diff
@@ -1,45 +1,16 @@
-variable "location" {
-    description = "The Azure Region in which all resources in this example should be created."
-    type        = string
-    default     = "usgovarizona"
-  
-}
+ 
 variable "create_resource_group" {
```

Comment:
You may have to update this variable. In Scaffolding, the variable is create_aks_resource_group